content/applications/finance/accounting/taxation/taxes/fiscal_positions.rst

```diff
@@ -28,19 +28,30 @@ Configuration
 
  .. _fiscal_positions/mapping:
 
-Tax and Account Mapping
+Tax and account mapping
 -----------------------
 
-To edit or create a Fiscal Position, go to :menuselection:`Accounting --> Configuration --> Fiscal
-Positions`, and open the entry you want to modify or click on *Create*.
+To edit or create a Fiscal position, go to :menuselection:`Accounting --> Configuration --> Fiscal
+Positions`, and open the entry you want to modify or click on :guilabel:`New`.
 
 The mapping of taxes and accounts is based on the default taxes and accounts defined in the
 products' forms.
```

Comment:
Soo, I'm not 100% sure I get your comment. A link can indeed be added regarding the info on taxes in product forms though I checked, and there is no good doc that directly refers to the product form. The two you put are related to general taxes but not in product form. 

For when you say there are three types, there is only one "customer taxes", you only see the two others because the localization is American.
So I'll look again for a nice link to add but I think the rest is not necessary.